jenkins/ci.suse.de/deploy-crowbar-poc

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
is there a reason why this is no longer moved here: `scripts/jenkins/crowbar/deploy-crowbar.sh` ??